codebleu/codebleu.py

```diff
@@ -7,7 +7,7 @@
 from . import bleu, dataflow_match, syntax_match, weighted_ngram_match
 
 PACKAGE_DIR = Path(__file__).parent
-AVAILABLE_LANGS = ["java", "javascript", "c_sharp", "php", "c", "cpp", "python", "go", "ruby"]  # keywords available
+AVAILABLE_LANGS = ["java", "javascript", "c_sharp", "php", "c", "cpp", "python", "go", "ruby", "rust"]  # keywords available
```

Comment:
Could you also add "rust" to
1. README.md
2. evaluate_app/README.md
2. evaluate_app/codebleu.py - description